version.sh

```diff
@@ -0,0 +1,4 @@
+PACKAGE_VERSION=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g' | tr -d '[[:space:]]')
```

Comment:
what about writing `version_fix.js` and then you can just `require('package.json').version` ?

Reply:
yeah.. that would work.. but then execute `sed` from within node or use node file i/o to update `package.bzl`?